windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -19,6 +19,18 @@ You must have a subscription to receive support for the Red Hat WMCO. Deploying
 
 include::modules/wmco-prerequisites.adoc[leveloffset=+1]
 
+[id="wmco-2-0-3"]
+== Release notes for Red Hat Windows Machine Config Operator 2.0.3
+
+Issued:2021-07-28
```

Comment:
I know the other entries are like this so it's fine to leave it. But I'm not sure why there isn't a space after the colon here (just something to consider in the future)